### PR TITLE
[FIX] Fix newline auto-generated

### PR DESCRIPTION
**Pull request by josep-tecnativa**
With the last change, updating docker-mailserver, when common.yaml is generated empty newline is generated.
CC @Tecnativa TT46621

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 
 def _load_copier_conf():
-    """Load copier.yml."""
+    """Load copier.yml ."""
```
Comment (pedrobaeza) [inline tasks.py:19]: Incorrect space

**Comment by pedrobaeza** [comment]
@josep-tecnativa you have to fix this. And please also check the warning:

```
tasks.py:131: [W8138(except-pass), write_code_workspace_file] pass into block except. If you really need to use the pass consider logging that exception
```

**Comment by pedrobaeza** [comment]
@josep-tecnativa this is still pending

**Comment by josep-tecnativa** [comment]
This problem is probably related with this commit:
https://github.com/Tecnativa/docker-duplicity/commit/510f54da8dc95fa5cecc4abba79ef9c4073f4945#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R126
I'm going to fix that.

**Comment by pedrobaeza** [comment]
Release done. Relaunch.

**Comment by josep-tecnativa** [comment]
> Release done. Relaunch.

Done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**